src/engine/engine.ts

```diff
@@ -85,10 +85,15 @@ function prepareOptions(origOptions: Schema, logger: logging.LoggerApi) {
     logger.info('Dry-run: No changes are applied at all.');
   }
 
-  return options;
+  return options as Schema;
 }
 
-function getOptionsString(options: Schema) {
+function getOptionsString(options: {
```

Comment:
Really good catch 🧠 

You gave an idea to create an Interface only with the params used by `npm publish`, which would be something like this `Pick<Schema, 'access' | 'tag' | 'otp' | 'dryRun'>`, this should be applied to the function `extractOnlyNPMOptions` in this same file

Reply:
Nice suggestion, will do it.